### ISSUE TITLE
Thumbnail considered a conflict

### ISSUE THREAD

**Issue by Alamandir** [WojciechKrysiak/SCModManager#45]
Would it be possible to somehow have the conflict checker skip over any instance of "thumbnail.jpg/png"? Would make the conflict checking massively more useful.

**Comment by matshou** [comment]
I agree, having the ability to filter or whitelist certain files would be really useful.

**Comment by matshou** [comment]
After poking around the application I've found it actually has a whitelist feature which can be accessed from "PreferencesWindow" that is opened by clicking on the 'cog' icon in the upper right corner. Just add filenames that should be ignored from conflict assessment and they will be ignored. Note that regex unfortunately does not work, but that's fine.
